Dockerfile

```diff
@@ -19,5 +19,14 @@ RUN abuild-keygen -i -a -n
 RUN apk update
 
 ADD src/aports /home/build/aports
+
+RUN if [ "${TARGETARCH}" = "arm64" ]; then pkg_arch=aarch64; else pkg_arch=x86_64; fi && \
```

Comment:
Just to be clear, this means all other editions that install these packages will end up with the newer versions. So even though this is only needed for Rancher desktop, if you install cni-plugins in alpine-lima, or std, then this will be the version used.

Reply:
> So even though this is only needed for Rancher desktop

This is not just needed for Rancher Desktop; it is needed for nerdctl to be able to [isolate bridge networks](https://github.com/containerd/nerdctl/blob/3ed9238f7f048eda9901e3047629610bd971cf9e/README.md#install).

So yes, this is intentional, but temporary. Once we move to Alpine 3.16 (when it is released), we will no longer install the `cni-plugins` from the edge repo.